src/transactions/test/InvokeHostFunctionTests.cpp

```diff
@@ -1886,29 +1900,35 @@ TEST_CASE("contract storage", "[tx][soroban]")
         {
             // Failure: contract data isn't in footprint
             test.putWithFootprint("key", ContractDataDurability::PERSISTENT, 88,
-                                  test.getContractKeys(), {}, false);
+                                  test.getContractKeys(), {},
+                                  INVOKE_HOST_FUNCTION_TRAPPED);
             test.getWithFootprint("key", ContractDataDurability::PERSISTENT,
-                                  test.getContractKeys(), {}, false);
+                                  test.getContractKeys(), {},
+                                  INVOKE_HOST_FUNCTION_TRAPPED);
             test.hasWithFootprint("key", ContractDataDurability::PERSISTENT,
-                                  test.getContractKeys(), {}, false);
+                                  test.getContractKeys(), {},
+                                  INVOKE_HOST_FUNCTION_TRAPPED);
             test.delWithFootprint("key", ContractDataDurability::PERSISTENT,
```

Comment:
Nit: We might want to add the error code interface to delete as well.

Reply:
I didn't do this because we never pass in false.